app/entities/user.go

```diff
@@ -13,18 +13,25 @@ type User struct {
 	LastName  string `json:"last_name"`
 	Password  string `json:"password"`
 	CreatedAt string `json:"created_at"`
+	DeletedAt string `json:"deleted_at"`
+}
+
+type UserID struct {
```

Comment:
There is no sense in having this struct. 
Why don't you use just a userID as a string in places where it's needed?

Reply:
Fair. Fixed